app/Repository/Eloquent/ClientRepository.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Repository\Eloquent;
+
+use App\Models\Client;
+use App\Repository\ClientRepositoryInterface;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Collection;
+
+class ClientRepository implements ClientRepositoryInterface
+{
+
+    protected $model;
+
+    public function __construct(Client $model)
+    {
+        $this->model = $model;
+    }
+
+    public function all(): Collection
+    {
+        return $this->model->all();
+    }
+
+    public function createClient(array $data): bool
+    {
+        return $this->model->create($data) ? true : false;
```

Comment:
Ambos métodos create y update si por alguna razón no se completan (a no ser que haya una excepción) retornarán false, así que la condición ternaria es innecesaria.
Así mismo, digamos que necesitas mostrar/devolver el record (en este caso Client) creado/actualizado, si lo dejas que retorne true, tendrías qué hacer otra consulta para obtenerlo. 
Es mejor dejarlo así: `$this->model->create($data)`

Actualiza todos tus repositorios para eliminar esta condición.

Reply:
Enterado